### ISSUE TITLE
CodeReviewer Quality Estimation (cls) Task

### ISSUE THREAD

**Issue by chiayi-hsu** [microsoft/CodeBERT#330]
Hello there, 

I'm wondering if I can **directly use a pre-trained model from Hugging Face for a Quality Estimation task**. When I use it and evaluate the model on your test set (cls-test.jsonl), the results are shown below. Compared with the paper, they seem lower. Therefore, I think I might have made a mistake in my evaluation. Could you share some suggestions?

```
 precision    recall  f1-score   support

           0     0.5041    0.8795    0.6409     15626
           1     0.5269    0.1343    0.2140     15617

    accuracy                         0.5070     31243
   macro avg     0.5155    0.5069    0.4274     31243
weighted avg     0.5155    0.5070    0.4275     31243
```

06/19/2025 20:23:11 - INFO - __main__ -   Target positive percentage: 0.49985596773677304

**Comment by chiayi-hsu** [comment]
> Hello there,
> 
> I'm wondering if I can **directly use a pre-trained model from Hugging Face for a Quality Estimation task**. When I use it and evaluate the model on your test set (cls-test.jsonl), the results are shown below. Compared with the paper, they seem lower. Therefore, I think I might have made a mistake in my evaluation. Could you share some suggestions?
> 
> ```
>  precision    recall  f1-score   support
> 
>            0     0.5041    0.8795    0.6409     15626
>            1     0.5269    0.1343    0.2140     15617
> 
>     accuracy                         0.5070     31243
>    macro avg     0.5155    0.5069    0.4274     31243
> weighted avg     0.5155    0.5070    0.4275     31243
> ```
> 
> 06/19/2025 20:23:11 - INFO - **main** - Target positive percentage: 0.49985596773677304
